manim/animation/creation.py

```diff
@@ -563,15 +563,22 @@ def __init__(
         **kwargs,
     ) -> None:
         self.time_per_char = time_per_char
+
+        # Check for empty text using family_members_with_points()
+        if not text.family_members_with_points():
+            raise ValueError(
+                "The input text is empty i.e., input has no characters to read thus cannot render a video file."
```

Comment:
Personally I'd prefer a shorter error message (e.g., `f"The text mobject {text} does not seem to contain any characters."`) and remove the comment about the video file (which is a consequence the user does not strictly need to know about).

Reply:
Okay got it Ben!